### PR TITLE
glossary check  and nicer(?) glossary javascript page

### PR DESCRIPTION
**Pull request by womullan**
Perhaps the try except could be wider - plus @timj may not like that i catch report and throw the exception on ... I guess I could stuff the message into a new exception ..

### CODE REVIEW

File: bin/generateAcronyms.py [hunk 1]
```diff
@@ -122,10 +121,16 @@ def read_glossarydef(filename, utags, init=None):
...
-            tags = row[ind + 2]
-            entryType = row[ind + 5]
+            try:
+                acr = row[ind]
+                defn = row[ind + 1]
+                tags = row[ind + 2]
+                entryType = row[ind + 5]
+            except BaseException as ex:
+                print("Error reading {} on line {} - {}".format(filename, lc, row),
+                      sys.exc_info()[0])
```
Comment (timj) [inline bin/generateAcronyms.py:131]: I'm not sure what the intent is here? Doesn't this print something like "<class 'IndexError'>" ?

I think if you want the class name reported you can do that with `type(ex).__name__`. You will already get the IndexError from reraising in the next line. Maybe this was a debug leftover?

File: bin/generateAcronyms.py [hunk 2]
```diff
@@ -465,16 +470,21 @@ def forceConverge(prevCount, utags):
         prevCount = count
 
 
+def setup_paths():
+    defaults_dir = os.path.join(
+        os.path.dirname(__file__), os.path.pardir, "etc")
+    lsst_glossary_path = os.path.join(defaults_dir, "glossarydefs.csv")
+    global_skip_path = os.path.join(defaults_dir, "skipacronyms.txt")
+    return [lsst_glossary_path, global_skip_path]
```
Comment (timj) [inline bin/generateAcronyms.py:478]: Return a tuple here and not a list.

File: bin/generateAcronyms.py (hunk 1 above)
Comment (womullan) [reply bin/generateAcronyms.py:131]: in deed it prints -- Error reading /Users/womullan/LSSTgit/lsst-texmf/bin/../etc/glossarydefs.csv on line 11 - ['ACCS', 'Auxiliary Camera Control System,LSST DM"', '', '', 'A'] <class 'IndexError'>
I should remove the IndexError part all I wanted was the line  number and entry

File: bin/generateAcronyms.py (hunk 2 above)
Comment (womullan) [reply bin/generateAcronyms.py]: ok

**Pull request merged**